### ISSUE TITLE
LU: SERVICE_DOWN

### ISSUE THREAD

**Issue by cliovincent** [schildbach/public-transport-enabler#337]
Since at least a week no public transport data from Luxembourg are available, e.g.:
### Query
- NetworkId: `LU`
- From: `Location{STATION, 160904011, 49.7518420/6.1103280, place=Mersch, name=Gare, products=[HIGH_SPEED_TRAIN]}`
- Via: `null`
- To: `Location{STATION, 200405035, 49.6006250/6.1337450, place=Luxembourg, name=Gare Centrale, products=[HIGH_SPEED_TRAIN, BUS]}`
- Date: `Thu Oct 08 08:35:05 GMT+02:00 2020`
- Departure: `true`
- Products: `[HIGH_SPEED_TRAIN, REGIONAL_TRAIN, SUBURBAN_TRAIN, TRAM, BUS]`
- Optimize for: `LEAST_DURATION`
- Walk Speed: `FAST`

### Error
```
De leveranciersdienst is momenteel uit de lucht :(
Probeer het later opnieuw.
```

### Additional information
[Please modify this part]

**Comment by ialokim** [comment]
It seems they changed their HAFAS endpoint [from `https://travelplanner.mobiliteit.lu/hafas/`](https://github.com/schildbach/public-transport-enabler/blob/master/src/de/schildbach/pte/LuProvider.java#L32) to `https://cdt.hafas.de/bin/mgate.exe` (see network requests at https://www.mobiliteit.lu/de/reiseplaner/).

A fix might be as easy as changing the server url, but needs testing.

**Comment by schildbach** [comment]
Fixed via 3154b34df5d4ea102e0ae8167fa368a33d91f4ca.

**Comment by ialokim** [comment]
Thanks a ton!
